assets/javascripts/discourse/initializers/discourse-automation.js

```diff
@@ -29,6 +34,136 @@ function _initializeDiscourseAutomation(api) {
 
     api.cleanupStream(_cleanUp);
   }
+
+  api.modifyClass("controller:create-account", {
+    pluginId: "discourse-automation",
+    customFields: {},
+    performAccountCreation() {
```

Comment:
Can you describe a bit why this override is needed? This is a risky codepath, the plugin would be overriding the account creation flow in core and if core changes in the future, there could be a constly regression here (i.e. users may not be able to create accounts, worst case scenario).